compiler/luci/pass/src/RemoveRedundantTransposePass.cpp

```diff
@@ -118,11 +115,10 @@ bool RemoveRedundantTransposePass::run(loco::Graph *g)
   bool changed = false;
   for (auto node : loco::active_nodes(loco::output_nodes(g)))
   {
-    auto circle_node = loco::must_cast<luci::CircleNode *>(node);
-    if (remove_consecutive_transpose_function(circle_node))
+    if (auto transpose = dynamic_cast<luci::CircleTranspose *>(node))
     {
-      changed = true;
-      break;
```

Comment:
@struss , did you have any problem to add a `break` here?

Reply:
Yes, but I think that `break` make this pass only update once for `RedundantTranspose` case. I think removing this nice.

Reply:
> Yes, 

What problem did you have?

Reply:
> > Yes,
> 
> What problem did you have?

Sorry to miss reply. I think removing that break make no problem on it. I first thought only one change would okay, so I added. but think about more deeply, this change could be done multiple. so I think removing that `break` is nice change.

> @struss , did you have any problem to add a `break` here?

Sorry for confusing. to reply on yes. I have no problem on this. Sorry.